src/views/Home/components/CLWCreditsDaysLeft/index.js

```diff
@@ -107,11 +105,15 @@ const CLWCreditsDaysLeft = () => {
               {`${daysRemaining} Days Left`}
             </Typography>
           </Grid>
-          <Grid item>
-            <Typography variant="body2" style={{ color: 'gray', textAlign: 'center' }}>
-              {`${remaining} CL&W Credit` + (remaining === 1 ? '' : 's') + ' Left'}
-            </Typography>
-          </Grid>
+          {required ? (
+            <Grid item>
+              <Typography variant="body2" style={{ color: 'gray', textAlign: 'center' }}>
+                {`${remaining} CL&W Credit` + (remaining === 1 ? '' : 's') + ' Left'}
+              </Typography>
+            </Grid>
+          ) : (
+            ''
+          )}
```

Comment:
Is there a reason to return an empty string instead of null here?
```suggestion
          ) : null
          }
```

Reply:
Force of habit